fleetspeak/src/server/https/https.go

```diff
@@ -138,7 +148,11 @@ func NewCommunicator(p Params) (*Communicator, error) {
 			ReadHeaderTimeout: 10 * time.Second,
 			WriteTimeout:      20 * time.Minute,
 			IdleTimeout:       30 * time.Second,
-		},
+		}
+	}
+	h := Communicator{
+		p: p,
+		hs: httpServer,
```

Comment:
http.Server should not be copied. You can work around this by creating a communicator with an httpServer as before, but without the TSLCfonfig. Then after creating the communicator, you can check the frontend config and add the TLSConfig if needed.

Reply:
Thanks a lot, Torben.
Updated as proposed.